### PR TITLE
Attempt to fix the put_record method.

### PR DESCRIPTION
**Pull request by Natim**
`requests-hawk` doesn't handle the json parameter.
For now we are using the old way of doing JSON requests.

### CODE REVIEW

File: syncclient/client.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 from hashlib import sha256
 from binascii import hexlify
+import json
+import six
```
Comment (almet) [inline syncclient/client.py:4]: I do not see six in the dependencies.

File: syncclient/client.py [hunk 2]
```diff
@@ -249,10 +253,19 @@ def put_record(self, collection, record, **kwargs):
         Note that the server may impose a limit on the amount of data
         submitted for storage in a single BSO.
         """
+        if isinstance(record, six.string_types):
```
Comment (almet) [inline syncclient/client.py:257]: We should add a note here stating that this shouldn't last once requests-hawk support for the JSON parameter is fixed.

**Comment by almet** [comment]
Looks like a good solution while we wait for support for JSON in requests-hawk.

File: syncclient/client.py (hunk 1 above)
Comment (Natim) [inline syncclient/client.py:4]: Added here: https://github.com/mozilla-services/syncclient/pull/17/files#diff-2eeaed663bd0d25b7e608891384b7298R17

**Pull request merged**